python/ray/tests/test_runtime_env_conda_and_pip.py

```diff
@@ -195,5 +196,17 @@ def test_import(self):
         wait_for_condition(lambda: check_local_files_gced(cluster), timeout=30)
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Don't support pip on Windows.")
```

Comment:
Probably needs linting if you accept my suggestion
```suggestion
@pytest.mark.skipif(sys.platform == "win32", reason="_PathHelper.get_virtual_activate not supported on Windows.")
```

Reply:
Addressed!